systemtest/src/main/java/io/strimzi/systemtest/Resources.java

```diff
@@ -481,8 +481,11 @@ private Kafka waitFor(Kafka kafka) {
         LOGGER.info("Waiting for Kafka pods");
         StUtils.waitForAllStatefulSetPodsReady(KafkaResources.kafkaStatefulSetName(name), kafka.getSpec().getKafka().getReplicas());
         LOGGER.info("Kafka pod are ready");
-        LOGGER.info("Waiting for Entity Operator pods");
-        StUtils.waitForDeploymentReady(KafkaResources.entityOperatorDeploymentName(name));
+        // EO should not be deployed if it does not contain UP and TO
```

Comment:
Should this comment be this instead?

```suggestion
        // EO should not be deployed if it does not contain UO or TO
```

Reply:
Fixed typos UP -> UO. But I don't agree to change `and` to `or` because of the following text in Strimzi documentation. 

> When both topicOperator and userOperator properties are missing, the Entity Operator will be not deployed.

Reply:
Right, sorry. The `and` is of course correct.